packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -488,13 +488,15 @@ export default defineComponent({
             alignH: 'right',
             wrap: 'nowrap'
           },
-          {
-            name: 'tags',
-            title: this.$gettext('Tags'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap'
-          },
+          this.capabilities.files.tags
+            ? {
+                name: 'tags',
+                title: this.$gettext('Tags'),
+                type: 'slot',
+                alignH: 'right',
+                wrap: 'nowrap'
+              }
+            : {},
```

Comment:
is empty object problematic?

Reply:
No, tested, its fine :+1:

Reply:
Thought about it again, cleanest solution would be to have the tags always in the fields model, because that's how the ResourceTable works (only renders the fields that are present in the data items or that are whitelisted). In `buildResources` you could add the `tags` attribute conditionally, not based on the capability, but based on whether or not the raw data has the dav property as key. The location that requests the data can omit or include the tags dav property, depending on whether or not the capabilities include the tags capability.

Reply:
-> following pr